### ISSUE TITLE
[FEATURE] - Event Notes

### ISSUE THREAD

**Issue by n3tsurge** [reflexsoar/reflex-api#190]
Allow an analyst to put notes and have conversations back and forth on a single event

**Comment by rediasec** [comment]
I would find it useful to be able to assign events to a user in ReflexSOAR just so others are aware that someone is already looking at that item and not duplicating work.

**Comment by n3tsurge** [comment]
We've added event comments and the ability to set an Event as Open in the next release.  Open status would be considered under investigation, does this suffice?

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by rediasec** [comment]
I understood, great then! It would suffice for some use cases such as auto creating a case for each High/Critical alert.

Thanks for the anwers!

**Comment by n3tsurge** [comment]
I agree now that I read this again the Open/Acknowledged event should have a "current assignee/owner" assigned, I'll make this happen

**Comment by rediasec** [comment]
Thank you very much for considering it! :pray:
